src/wavesurfer.js

```diff
@@ -1290,6 +1290,9 @@ export default class WaveSurfer extends util.Observer {
      * );
      */
     load(url, peaks, preload, duration) {
+        if (url === undefined) {
+            throw new Error('url parameter is invalid');
```

Comment:
Maybe `if (!url)` to also throw if it's an empty string or null?
The error message could then be `The URL parameter cannot be empty` or similar.

Reply:
sounds good, was struggling to find a good description for the error but that's a better check.